Local build works ok, but CodeShip has a bad day.
```
2017-11-08T18:55:43.939Znpm ERR! code ECONNRESET
npm ERR! errno ECONNRESET
npm ERR! network request to https://registry.npmjs.org/chalk/-/chalk-1.1.3.tgz failed, reason: socket hang up
npm ERR! network This is a problem related to network connectivity.
npm ERR! network In most cases you are behind a proxy or have bad network settings.
npm ERR! network 
npm ERR! network If you are behind a proxy, please make sure that the
npm ERR! network 'proxy' config is set properly.  See: 'npm help config'
```

@biern Can you check clean `npm install` from your computer?

--- a/docker/base/Dockerfile
+++ b/docker/base/Dockerfile
@@ -1,4 +1,4 @@
-FROM node:8.5
+FROM node:8.9
 
 # Truffle dependencies
 RUN apt-get -y update &&\
@@ -9,5 +9,5 @@ RUN mkdir -p /app
 WORKDIR /app
 
 ADD / .
-RUN rm -fr node_modules
+RUN rm -rf node_modules
 RUN npm install

